Sensus.Tests.Cloud/Sensus.Tests.Cloud.csproj

```diff
@@ -70,11 +70,5 @@
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
       <Private>False</Private>
     </ProjectReference>
-    <ProjectReference Include="..\Sensus.Android\Sensus.Android.csproj">
-      <Project>{D33B6CDB-795C-4826-B131-47C526ACE07F}</Project>
-      <Name>Sensus.Android</Name>
-      <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
-      <Private>False</Private>
-    </ProjectReference>
   </ItemGroup>

```

Comment:
@MatthewGerber Sorry about that. I'm working on some tests for the callback system, but they're not commit-ready yet. I must have thought this was related to them.

On that note, I'm meeting with Mark to ask a few questions re the tests on Wednesday, and I'm hoping to have them added here by the end of the week.